### ISSUE TITLE
Print line breaks in tooltips?

### ISSUE THREAD

**Issue by WrinklyNinja** [loot/loot#390]
As text in various languages varies in length, it could be useful for translators to be able to control the sizes of tooltips to prevent clipping issues, by including line breaks where necessary in their translated text. This would require the tooltip element to be styled with `white-space: pre-line;` or similar, and there may need to be adjustments to line height, etc.

**Comment by WrinklyNinja** [comment]
Tooltip text (that isn't in a `<div tip>`) gets inserted as a text node within `<core-tooltip>`'s `<content select="[tip]">` element. The content element's special role seems to mean the text can't be styled: none of the CSS I tried had any effect, at least. 

I tried to style the content element's parent with `white-space: pre-line;`, but that just caused all the white-space to be treated as line breaks. Using the `white-space: pre;` styling stopped that happening, but there's weird indenting of the first line.

Putting the text within a `<span>` within the `<content>` worked, so a `<div tip>` would work, but then I had a large top margin.

In summary: problems, problems.

**Comment by WrinklyNinja** [comment]
I figured out what the problem is: I can't have text wrapping by default because this is disabled using `white-space: nowrap` default styling. This is done because the `<core-tooltip>` element width can be much narrower than the actual tooltip child's width needs to be (eg. tooltip for an icon), and so without the `nowrap`, text wraps to try and fit in that parent width. 

The `nowrap` effectively makes the tooltip overflow the `<core-tooltip>` area, and since tooltip sizing varies according to content, I can't set tooltip width to override the parent width without JS.

**Comment by WrinklyNinja** [comment]
This is no longer an issue with Polymer 1.0's tooltips, closing this in favour of tracking #434.
